src/frontend/js/order.js

```diff
@@ -3,6 +3,35 @@ $(function() {
     // Javascript of the order application.
     // **************************************
 
+    if ($('.ui.order-override.modal')) {
+      $('.ui.order-override.modal').modal('show');
```

Comment:
@aaboffill Agreed. I will make the necessary changes in the pull.

@lingxiaoyang Good idea - ultimately I think we should revisit the JS for this project from the ground up. I will create an issue for further discussion.